engine/controllers/command_line_parser.cc

```diff
@@ -150,6 +151,18 @@ bool CommandLineParser::SetupCommand(int argc, char** argv) {
     mdc.Exec(model_id);
   });
 
+  std::string model_alias;
+  auto model_alias_cmd =
+      models_cmd->add_subcommand("alias", "Add alias name for a modelID");
+  model_alias_cmd->add_option("--model_id", model_id, "Can be modelID or model alias to identify model");
+  model_alias_cmd->require_option();
+  model_alias_cmd->add_option("--new_model_alias", model_alias, "new alias to be set");
```

Comment:
@nguyenhoangthuan99 Small nitpick: I recommend we simplify this to just `--alias`

Reply:
I'll update it